File: MyPlanner.Clients/React/src/entities/TodoList.ts
import TodoTask from "./TodoTask";

export default class TodoList{
    id : string;
    title: string;
    tasks: TodoTask[];
    constructor(id: string, title: string, tasks : TodoTask[] = []){
        this.id = id;
        this.title = title;
        this.tasks = tasks;
    }
}

File: Backends/DotNet/MyPlanner.Data/Entities/Todo/TodoList.cs
using MyPlanner.Data.Entities.Common;

namespace MyPlanner.Data.Entities.Todo;

public class TodoList : PageContent
{
    public TodoList(Guid pageId) : base(pageId)
    {
        Type = PageContentEnum.TodoList;
    }
    public List<TodoTask> Tasks { get; set; } = new List<TodoTask>();
}
